docs/docs-content/vm-management/create-vmo-profile.md

```diff
@@ -165,12 +189,76 @@ profile configuration. To learn about pack components, refer to [Palette VMO](./
 
 </TabItem>
 
+<TabItem value="airgap" label="Airgap">
+
+1. Log in to Palette using your configured system address.
```

Comment:
```suggestion
1. Log in to a tenant that belongs to your instance of Palette or Palette VerteX.
```

Reply:
I feel like this is more specific, since system credentials could confuse people into thinking they should log in to the system console :eye: